app/src/main/java/com/github/se/cyrcle/model/address/Address.kt

```diff
@@ -0,0 +1,80 @@
+package com.github.se.cyrcle.model.address
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Represents an address.
+ *
+ * @property publicName The public name of the address. This can be for example the name of a
+ *   hospital or a park.
+ * @property house The house number or name.
+ * @property road The road name.
+ * @property cityBlock The city block, residential, farm, farmyard, industrial, commercial or
+ *   retail.
+ * @property neighbourhood The neighbourhood, quarter or allotments.
+ * @property hamlet The hamlet, isolated dwelling or croft.
+ * @property suburb The suburb, city district, district, borough or subdivision.
+ * @property city The city, town, municipality or village.
+ * @property postcode The postcode.
+ * @property region The region, state, state district, county or ISO3166-2-lvl.
+ * @property country The country.
+ * @property continent The continent.
+ */
+data class Address(
+    @SerializedName(
+        "amenity",
+        alternate =
+            [
+                "emergency",
+                "historic",
+                "military",
+                "natural",
+                "landuse",
+                "place",
+                "railway",
+                "man_made",
+                "aerialway",
+                "boundary",
+                "aeroway",
+                "club",
+                "craft",
+                "leisure",
+                "office",
+                "mountain_pass",
+                "shop",
+                "tourism",
+                "bridge",
+                "tunnel",
+                "waterway"])
+    val publicName: String = "",
+    @SerializedName("house_number", alternate = ["house_name"]) val house: String = "",
+    @SerializedName("road") val road: String = "",
```

Comment:
For attributes like these, is it necessary to add a `SerializedName`? I see some that don't have any and some with, even though there is only one name

Reply:
It's not necessary, but I figured it would be nice for consistency (e.g. if we decided to change the variable `road` into `street`, and also if some have a serializedName i figured it would "look" better if all did). Also I don't understand what you mean by "some that don't have them"? I triple checked and it seems that all have one

Reply:
Right my bad, they all have them 😅